src/data/markdown/docs/01 guides/02 Using k6/08 Tags and Groups.md

```diff
@@ -43,6 +44,37 @@ execution of the supplied `group()` function also emits a `group_duration`
 function. This, combined with the metric tags described below, enables very flexible performance
 monitoring of different groups in your test suite.
 
+**Group anti-pattern**
+
+It is often an anti-pattern to wrap each individual request within a group.
+
+<CodeGroup labels={["group-antipattern.js"]} lineNumbers={[true]}>
+
+```javascript
+// reconsider this type of code
+group('get post', function () {
+   http.get(`http://example.com/posts/${id}`);
+});
+group('list posts', function () {
+   let res = http.get(`http://example.com/posts`);
+   check(res, {
+     'is status 200': (r) => r.status === 200,
+   });
+});
+```
+
+</CodeGroup>
+
+Groups were designed to help you visualize the various parts of a large test. 
+If your code looks like the example above, consider the following alternatives:
+
+- For dynamic URLs, use the [URL grouping feature](/using-k6/http-requests#url-grouping).
+- To provide a meaningful name to your request, set the value of [tags.name](/using-k6/http-requests#http-request-tags).
+- To reuse common logic or organize your code better, create a [local Javascript module](/using-k6/modules#local-filesystem-modules) and import it into the test script.
+
```

Comment:
Maybe link to the scenarios page here as well?

Reply:
agree

Reply:
solved by https://github.com/loadimpact/k6-docs/pull/210/commits/aa00ae948aa7739e50c4b726e156ec78a0c3ec18